Server/src/app.module.ts

```diff
@@ -5,9 +5,14 @@ import { AppService } from './app.service';
 import { LocationModule } from './location/location.module';
 import config from 'ormconfig';
 import { Location } from './location/entities/location.entity';
-
+import { ServeStaticModule } from '@nestjs/serve-static';
+import { join } from 'path';
 @Module({
```

Comment:
This will be more readable if you group the imports, i.e.
1. Node built-ins
2. Frameworks
3. Filips own application code

```typescript
import { join } from 'path';

import { ServeStaticModule } from '@nestjs/serve-static';

import { Location } from './location/entities/location.entity';
import { LocationModule } from './location/location.module';
import config from 'ormconfig';
```

Please read the PEP8 advices on imports, https://peps.python.org/pep-0008/#imports Guido and Pythonistas knows about style and readability. 🙃